server/resolvers/TestPlanReportOperations/updateStatusResolver.js

```diff
@@ -12,7 +12,11 @@ const updateStatusResolver = async (
     { status: status },
     { user }
 ) => {
-    if (!user.roles.includes('ADMIN')) {
+    let roles = [...user.roles];
```

Comment:
@alflennik seems there's 2 structures of the user roles coming through to this resolver.

`[ 'ADMIN' ... ]` and `[ { name: 'ADMIN' }, ... ]`

Currently accounting for both and this isn't a blocker to the PR but investigation may be required. I've went ahead and copied this logic to other resolvers dependent on the user role as well.

Reply:
Thanks for finding this - sounds like you were able to fix it. Thank you!